packages/frontend/src/components/composer/ComposerMessageInput.tsx

```diff
@@ -46,10 +46,9 @@ export default class ComposerMessageInput extends React.Component<
     this.insertStringAtCursorPosition =
       this.insertStringAtCursorPosition.bind(this)
 
-    this.saveDraft = debounce(() => {
-      const { text, chatId } = this.state
+    this.saveDraft = throttle((text: string, chatId: number) => {
       this.props.updateDraftText(text.trim() === '' ? '' : text, chatId)
-    }, 1000)
+    }, 200)
```

Comment:
Is the timer supposed to be lowered still?

Reply:
I think this is sufficient at least in my tests. Somehow the saveDraft seems to be blocking anyway so if I click on a chat or account immediately after typing the click has no effect.
I still managed to have a missing character in the draft or one left over after deleting but I think that's ok. I had to type very fast and click mutiple times on another chat to achieve that.

Reply:
> so if I click on a chat or account immediately after typing the click has no effect.

Uhh, is this also on the 1.48 release version?

Reply:
> I think this is sufficient at least in my tests.

What I meant is that is it supposed to be lowered to 200 instead of staying at 1000. Because switching to `throttle` is supposed to be enough.

Reply:
> Uhh, is this also on the 1.48 release version?

Yes with current master.

Reply:
It even blocks my mouse pointer whe I try to move it
On current main
I enter fast 5-6 characters and then try to click somewhere  - it takes until the draft is saved before I can interact again